land.gemspec

```diff
@@ -1,56 +1,58 @@
-$:.push File.expand_path("lib", __dir__)
+$:.push File.expand_path('lib', __dir__)
```

Comment:
I would turn off this whitespace rubocop change. We don't wanna impose our stylistic guidelines on this public gem.